tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/RabbitMQ.cs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // See this project's .csproj file for target framework => RabbitMQ.Client version mappings
-#if NETCOREAPP3_1 || NET48 || NET5_0 || NET6_0
+#if NETCOREAPP3_1 || NET48 || NET5_0 || NET6_0 || NET7_0
```

Comment:
I don't think we need to itemize all the versions of .NET/.NET Core here anymore after dropping support for .NET Core 2.0/2.1/3.0
```suggestion
#if NETCOREAPP || NET48
```